### PR TITLE
Tutorial update: how to add cluster-specific information

### PR DESCRIPTION
**Pull request by skydoorkai**

### CODE REVIEW

File: docs/tutorials/elasticdl_on_prem_cluster.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ export TRAVIS_BUILD_DIR=$PWD
 bash scripts/travis/build_images.sh
```
Comment (QiJune) [inline docs/tutorials/elasticdl_on_prem_cluster.md]: Commands here need to be updated too. Or we plan to update it in another PR?

File: docs/tutorials/elasticdl_on_prem_cluster.md [hunk 2]
```diff
@@ -69,3 +69,48 @@ elasticdl train \
  --image_pull_policy=Always \
  --namespace=kubemaker
 ```
+
+## Add Cluster-specific Information
```
Comment (wangkuiyi) [inline docs/tutorials/elasticdl_on_prem_cluster.md]: specific => Specific

File: docs/tutorials/elasticdl_on_prem_cluster.md [hunk 3]
```diff
@@ -69,3 +69,48 @@ elasticdl train \
...
+
+If the on-prem cluster requires additional specifications to pods or services,
```
Comment (wangkuiyi) [inline docs/tutorials/elasticdl_on_prem_cluster.md]: on-prem => on-premise

Comment (wangkuiyi) [reply docs/tutorials/elasticdl_on_prem_cluster.md]: Usually, the on-premises clusters wouldn't require additional YAML labels.  It is the tailored version of Kubernetes who requires additional labels.

File: docs/tutorials/elasticdl_on_prem_cluster.md [hunk 4]
```diff
@@ -69,3 +69,48 @@ elasticdl train \
...
+such as labels, tolerations, etc, we can add an additional argument
```
Comment (wangkuiyi) [inline docs/tutorials/elasticdl_on_prem_cluster.md]: labels are for specifying fault tolerance-related specifications, it doesn't make sense to put labels and tolerance side-by-side.

File: docs/tutorials/elasticdl_on_prem_cluster.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ export TRAVIS_BUILD_DIR=$PWD
 bash scripts/travis/build_images.sh
 ```
 
-## Submit job to cluster
+## Submit Job to Cluster
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_on_prem_cluster.md:15]: The train command should be updated too.
We use --image_name directly.
--image_base and --docker_image_prefix are not necessary anymore.

File: docs/tutorials/elasticdl_on_prem_cluster.md (hunk 1 above)
Comment (skydoorkai) [reply docs/tutorials/elasticdl_on_prem_cluster.md]: Changed.

File: docs/tutorials/elasticdl_on_prem_cluster.md (hunk 2 above)
Comment (skydoorkai) [reply docs/tutorials/elasticdl_on_prem_cluster.md]: Changed.

File: docs/tutorials/elasticdl_on_prem_cluster.md (hunk 3 above)
Comment (skydoorkai) [reply docs/tutorials/elasticdl_on_prem_cluster.md]: Revised.

File: docs/tutorials/elasticdl_on_prem_cluster.md (hunk 4 above)
Comment (skydoorkai) [reply docs/tutorials/elasticdl_on_prem_cluster.md]: Revised.

File: docs/tutorials/elasticdl_on_prem_cluster.md (hunk 5 above)
Comment (skydoorkai) [reply docs/tutorials/elasticdl_on_prem_cluster.md:15]: Changed the submit script.

**Review by brightcoder01 — APPROVED** [review]

**Pull request merged**